File: ruby/tests/206-is_alive-dataprovider.rb
admin   = admin_handle

dp_id   = @dp_list_200.first[:id] || nil
raise "SKIPPED: no DataProviders" unless dp_id

is_alive = admin.is_alive_data_provider(dp_id)

puts "DP_IS_ALIVE: #{is_alive.inspect}" if @debug

raise "Value returned is not a boolean" unless (is_alive.is_a?(TrueClass) || is_alive.is_a?(FalseClass))

true

File: perl/tests/206-is_alive-dataprovider.pl
my $admin      = &AdminHandle;
our $dp_list_200;

die "SKIPPED: no Data Providers\n" if scalar(@$dp_list_200) == 0;

my $dp_id    = $dp_list_200->[0]->{'id'};
my $is_alive = $admin->is_alive_data_provider($dp_id);

if ($DEBUG) {
  my $dump_is_alive = Data::Dumper::DumperX($is_alive);
  print "DP_IS_ALIVE: $dump_is_alive\n";
}

die "Value returned not a boolean.\n" unless ($is_alive == 0 || $is_alive == 1);

1;
